themes/default/content/blog/deploy-aws-static-website-with-angular/index.md

```diff
@@ -0,0 +1,116 @@
+
+In Angular, the basic building blocks of a website are called _components_. These are reusable structures that contain their own logic and functionality. For example, you could create a button as an Angular component and program it to have a particular look and a particular behavior. Then, you would reuse the button component wherever you need it in the website. The button could also be configurable meaning that, based on different properties, you could change its look and behavior.
+
+A simple website, like the one we’ll be building in this post, is mainly created from a group of Angular components.
+
+## What is the AWS Static Website Component?
+
+The AWS Static Website component from Pulumi makes it easy to deploy an AWS S3 static website and, optionally, add a CloudFront content distribution network (CDN). While you can use any of the programming languages Pulumi supports (TypeScript, JavaScript, Python, Go, .NET, Java, and YAML), we’ll be using TypeScript, since that is what the Angular framework uses.
+
+The great advantage of the AWS Static Website component is that you can have a complete, functioning site in a few minutes. The component encapsulates all the complex logic that an AWS static website requires. Without the component, you would need to spend hours or even days writing and debugging code to get the same result.
+
+If you’re interested in learning more about the component, read [Deploy Static Websites to AWS using 10 lines of YAML]({{< relref "/blog/deploy-aws-static-website-with-yaml" >}}).
+
+## The Angular Components
+
+All the Angular components you need to build the website are available in the GitHub [repository](https://github.com/sean1588/angular-static-site-pulumi). The repository contains:
+
+* A component for the [left navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/leftnav).
+* A component for the [top navigation bar](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/shared/topnav).
+* A component for the [Home](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/home) page.
+* A component for the [Menu](https://github.com/sean1588/angular-static-site-pulumi/tree/main/src/app/components/menu) page.
+
+We also use some components that come with the Angular framework.
+
+Each of the components has a TypeScript file that contains the programming logic and an HTML file that describes how the component will render.
+
+## Building the Pulumi Café Website
+
+Here are the instructions to build the website.
+
+1. Clone the GitHub repository located [here](https://github.com/sean1588/angular-static-site-pulumi).
+1. Run `npm install` to install the website’s node dependencies.
+1. Run `npm build` to generate the static website files and place them in a directory called **dist**. We will reference this directory when we configure Pulumi to deploy the site.
```

Comment:
When I run this (with Node 14 now), I get:

```
npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?
```

`npm run build` does work, though.

Reply:
Could we also use `backticks` instead of **bold** for these literals? That'd be more consistent with how we normally quote them.

Reply:
thank you! yeah totally meant `node run build`.